File: Python/String-1/n_twice.py

# given a string and an int n, return a string made of the first and last n chars from the string
# the string length will be at least n
def n_twice(str, n):
    s = ''

    if len(str) >= n:
        s = str[:n] + str[-n:]

    return s

print(n_twice('Hello', 2))
print(n_twice('Chocolate', 3))
print(n_twice('Chocolate', 1))

File: JavaScript/String-1/nTwice.js

// given a string and an int n, return a string made of the first and last n chars from the string
// the string length will be at least n
const nTwice = (str, n) => {
    return str.substring(0, n) + str.substring(str.length - n);
};

console.log(nTwice('Hello', 2));
console.log(nTwice('Chocolate', 3));
console.log(nTwice('Chocolate', 1));
